### DISCUSSION TITLE
Discussing some bugs related to eccentrictouch

### DISCUSSION THREAD

**Discussion by ItsMeIshan** [DesignSystemsOSS/eccentrictouch#225] [Q&A]
I think there should be some way to remove the CSS that comes with CDN of the eccentrictouch because it overwrites the css that is already there in our project and then it gets difficult to overwrite the CSS or if there's some way to do this then it should be mentioned in the docs.

**Comment by wh0sumit** [comment]
@ItsMeIshan I think this was the main issue right now, also the name of the classes are too long.

**Reply by ItsMeIshan** [reply]
Yes @WH0SUMIT because this issue can ruin user experience and we would never want to that!

**Reply by ItsMeIshan** [reply]
Can we discuss how we will implement this if anyone is up with an idea?

**Reply by wh0sumit** [reply]
I think it is a simple and important issue basically in eccentric touch CSS code the default features of HTML elements are overwritten that's why we are not able to use the basic default features of HTML elements but if we can change the CSS files it will work fine.

**Reply by ItsMeIshan** [reply]
I think we can do that we should open a issue on this like "cannot use default features of html with the CDN"
